src/server/storage/storageAPI.js

```diff
@@ -3,20 +3,29 @@ import { Router } from 'express'
 import passport from "passport"
 import { get } from 'lodash'
 import { type StorageAPI } from '../../imports/types'
-import { wrapAsync } from '../server-middlewares'
+import { wrapAsync, onlyInProduction } from '../server-middlewares'
+import { sendOTP } from '../../imports/otp'
 
 const setup = (app: Router, storage: StorageAPI) => {
   app.post("/user/add", passport.authenticate("jwt", { session: false }), wrapAsync(async (req, res, next) => {
-    const { user, body } = req
-    await storage.addUser(body.user)
+    const { body: { user } } = req
+    await storage.addUser(user)
     res.json({ ok: 1 })
   }))
 
   app.post("/user/delete", passport.authenticate("jwt", { session: false }), wrapAsync(async (req, res, next) => {
-    const { user, body } = req
-    await storage.deleteUser(body.user)
+    const { body: { user } } = req
+    await storage.deleteUser(user)
     res.json({ ok: 1 })
   }));
-}
 
+  app.post("/user/mobile", passport.authenticate("jwt", { session: false }), onlyInProduction, wrapAsync(async (req, res, next) => {
```

Comment:
move this endpoint also to the verification. and use storage.updateUser (as commented before)
verify/sendotp